.github/workflows/seed.yml

```diff
@@ -99,11 +99,17 @@ jobs:
         env:
           FORCE_COLOR: "2"
         run: |
-          pnpm seed:local test --generator ruby-model
+          pnpm seed:local test --generators ruby-model
 
       - name: Ensure no changes to git-tracked files
         run: git --no-pager diff --exit-code  -- ":(exclude)seed/*/.mock/*"
 
+      # Only register if seed is passing
+      - name: Register on push to main
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+        run: |
+          pnpm seed:local register --generators ruby-model
```

Comment:
Talked offline, ideally we should have a separate workflow for `register-generator`.

Reply:
Stop throwing if no error
Remove register from seed.yml CI job
On merge to main:
- run publish
- run register

Reply:
Also update the publish job to use git show instead of clone twice